docs/api-reference/deckgl.md

```diff
@@ -119,5 +119,23 @@ object for the topmost picked layer at the coordinate
 are affected.
 - `event` - the original [MouseEvent](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent) object
 
+### Methods
+
+##### `queryObjects(boundingBox, layerId)`
+
+Get all pickable and visible objects within a bounding box.
+
+Parameters:
+- `boundingBox` (Object) - the bounding box in screen coordinates
+  + `x` (Number) - left of the box in pixels
+  + `y` (Number) - top of the box in pixels
+  + `width` (Number) - width of the box in pixels
```

Comment:
- Could `width` and `height` default to `1`
- Or should we have a separate `queryObject` that could have ({x, y, radius}) ?
- I am thinking the latter since `queryObject` is precise but `queryObjects` isn't.

- I generally try to go with flat named parameter lists. If we feel that the app is going to maintain boundingBoxes separately I guess the nesting could be worthwhile.